packages/components/radio/src/radio.vue

```diff
@@ -59,4 +59,10 @@ const { radioRef, radioGroup, focus, size, disabled, modelValue, actualValue } =
 function handleChange() {
   nextTick(() => emit('change', modelValue.value))
 }
+
+function handleClick() {
+  if (radioGroup?.clearable && modelValue.value === actualValue.value) {
+    modelValue.value = undefined
+  }
+}
```

Comment:
Duplicate of the above. It can be removed to `useRadio` hook.

Reply:
done